chiselFrontend/src/main/scala/chisel3/core/StrongEnum.scala

```diff
@@ -15,32 +14,41 @@ import firrtl.annotations._
 
 
 object EnumAnnotations {
-  case class EnumComponentAnnotation(target: Named, enumTypeName: String) extends SingleTargetAnnotation[Named] {
+  case class EnumComponentAnnotation(target: Named, typeName: String) extends SingleTargetAnnotation[Named] {
```

Comment:
Can you add a comment explaining what this, and the other annotations, are for?
The chisel ones can just refer to the firrtl ones